### ISSUE TITLE
Error with resulting image

### ISSUE THREAD

**Issue by WujiangXu** [xiaoyu258/GeoProj#3]
I first use the eval.py to generate the npy file.Then I use the resampling.py to get the resulting image.But the image edges are jagged and some of the pixels are missing. Points that lose pixels are replaced by white ones.
About the model,I use the pretrained model that you provided.

**Comment by sanahtech** [comment]
how we generate npy file from eval.py file? because it results only .mat files.

**Comment by xiaoyu258** [comment]
Mat files save a dictionary {'u': u,'v': v} which is the same as numpy array [u, v] with the shape of 2\*H\*W.

**Comment by WujiangXu** [comment]
> Mat files save a dictionary {'u': u,'v': v} which is the same as numpy array [u, v] with the shape of 2*H*W.

Whether the loss function in the trainNetS.py need to change?It seems to need to be divided by the batchsize

**Comment by xiaoyu258** [comment]
No need to change the loss function for GeoNetS. For the missing pixels, maybe you need to do the model fitting after GeoNetM as the paper do to smooth the flow. You can find an example in Figure 1 of Supplementary Material.

**Comment by WujiangXu** [comment]
Thank you for your reply.
1.If it is not divided by the "batchsize".I got the loss approximately 20.I use Validation images of the Data of Places365-Standard as my dataset.
2.Can we do the model fitting in your code？We need to find the  Hough transform parameter
by ourselves after training the model.
